app/src/main/res/layout-land/fragment_suggested_edits_add_descriptions_item.xml

```diff
@@ -84,58 +84,46 @@
                     android:ellipsize="end"
                     tools:text="Lorem ipsum"/>
 
-                <androidx.constraintlayout.widget.ConstraintLayout
+                <LinearLayout
```

Comment:
If you are using the `LinearLayout`, you should not put the related `ConstraitLayout` attributes. (e.g. `layout_constraintStart_toStartOf`)

But I would recommend to keep using the `ConstraintLayout` anyway.

Reply:
Done